### ISSUE TITLE
Peer Review of Final Report

### ISSUE THREAD

**Issue by CallumChalmers29** [kdea020/metabolomics-project#6]
Hi Kieran,

Just had a read of your final report, the stuff you have written so far is excellent, very easy to read and flows very nicely. Great job! At the moment there is a bit of technical detail e.g liquid and gas mass spec that could be a bit intimidating to the layperson so make sure to explain that in your intro :) Also in the reuse potential section you use the acronyms GC-MS and LC-MS for gas and liquid chromatography without having referenced the acronyms previously so maybe put that in the intro too. One last thing (which is not really to do with the report) but your license is CC-BY 3.0 NZ, would you consider updating this to CC-BY 4.0 International? It is essentially the same but is just easier to use across different jurisdictions. 

This report was seriously impressive so keep up the awesome work!
Callum

**Comment by CallumChalmers29** [comment]
@kdea020, the final report is due on Thursday, just checking to see, are you on track with the changes I suggested? :)

**Comment by kdea020** [comment]
Hey Callum! Thank you for the great feedback, especially the use of acronyms and making the introduction more accessible - something I didn't even consider, but likely very important since we're writing for a software journal!

Will try and put the changes up by tomorrow evening at latest.

**Comment by kdea020** [comment]
Hey Callum,

I'm sure you're hard at work on your own project, but I would greatly appreciate it if you have time to spare to look over the changes I've made to my (still incomplete haha) final report and judge if I've responded to your feedback well? Thanks :)

**Comment by CallumChalmers29** [comment]
Hey Kieran,

You have definitely made some good changes and the intro is shaping up nicely, you've addressed the minor acronym issue which is great and you have definitely used simpler terminology which is great. A couple of things I've noticed:
- Your abstract is quite long and gets a bit heavy - maybe make this a bit more general and move some of the details like gas and liquid chromatography and equipment noise into the intro
- In the intro, I think you should put the bit about metabolomics first and then go into the data issues rather than the other way round, this seems semantic but I think it flows nicer that way
- Also be sure to explain what metabolites and what the entire purpose of metabolomics, it will vastly improve the reuse potential of your software if people with no background in science can understand what your software does
- Finally, I noticed that your in text citations are APA, they should be the number in square brackets like this [1]

All the best with finishing your assignment, you've done some really great work!
Callum

**Comment by kdea020** [comment]
Thanks very much for further feedback! I agree with those points.

However, about the citations - I looked up Harvard referencing and it appears to use that format for in-line citations. (For example as described here http://www.otago.ac.nz/library/pdf/Harvard_referencing.pdf ). Zotero also agrees with me... were we instructed specifically to ignore that and use square brackets for in-text citations??

**Comment by CallumChalmers29** [comment]
Yeah if you have a look at the link to the final assignment template in slack (General channel) it says in text citations are square brackets (Sorry I just assumed this was the standard, silly me!)

**Comment by kdea020** [comment]
Ah okay thanks for the heads up! Missed that one haha
